tests/test_config_flow.py

```diff
@@ -29,7 +29,7 @@ async def test_flow_invalid_scan_interval_range(
         DOMAIN, context={"source": SOURCE_USER}
     )
 
-    with pytest.raises(voluptuous.error.MultipleInvalid):
+    with pytest.raises(data_entry_flow.InvalidData):
```

Comment:
```suggestion
    with pytest.raises((voluptuous.error.MultipleInvalid):
```

Shouldnt this be the error asserted as the value should be at least 15?

Reply:
sadly no, I tried this but the validation seems to have changed https://github.com/inverse/home-assistant-iolite-component/actions/runs/8663239816/job/23756953149?pr=31#step:5:214

maybe I'll just rollback the dependency update